src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    public static final String NULL_VALUE = null;
+    public static final String VALID_NAME = "Tom123";
+    public static final String VALID_PASSWORD = "aBc321";
+    public static final Integer VALID_AGE = 22;
+
+    private static RegistrationService registrationService;
+    private static User actualUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void userNotNullCheck() {
+        actualUser = new User(NULL_VALUE, VALID_PASSWORD, VALID_AGE);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualUser));
+    }
+
+    @Test
+    void userPassNotNullCheck() {
```

Comment:
```suggestion
    void user_nullPassword_notOk() {
```
check the namings for the test cases in the README file
same for all